concent_api/gatekeeper/views.py

```diff
@@ -121,73 +157,193 @@ def parse_headers(request: WSGIRequest, path_to_file: str) -> Union[FileTransfer
     # Check ConcentFileTransferToken each field:
     # -DEADLINE
     if not isinstance(loaded_golem_message.token_expiration_deadline, int):
-        return gatekeeper_access_denied_response('Wrong type of token_expiration_deadline variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Wrong type of token_expiration_deadline variable.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     current_time = get_current_utc_timestamp()
 
     if current_time > loaded_golem_message.token_expiration_deadline:
-        return gatekeeper_access_denied_response('token_expiration_deadline has passed.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'token_expiration_deadline has passed.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
```

Comment:
Situations like deadlines and token expiration should have separate codes because it's likely that the client will have to react to that specific situation in a different way that for other errors. For example if you have a wrong type somewhere, your token is completely broken. But if the time runs out, there's sometimes a mechanism to renew the token.